variables.mk

```diff
@@ -3,53 +3,88 @@
 #########################################################################################
 
 #########################################################################################
-# default variables to invoke the generator for a example Rocket system
+# variables to invoke the generator
 # descriptions:
-#   PROJECT = the scala package to find the MODEL/Generator in
+#   SBT_PROJECT = the SBT project that you should find the classes/packages in
 #   MODEL = the top level module of the project in Chisel (normally the harness)
 #   VLOG_MODEL = the top level module of the project in Firrtl/Verilog (normally the harness)
+#   MODEL_PACKAGE = the scala package to find the MODEL in
 #   CONFIG = the configuration class to give the parameters for the project
-#   CFG_PROJECT = the scala package to find the CONFIG class
-#   SBT_PROJECT = the SBT project that you should find the Generator class in
+#   CONFIG_PACKAGE = the scala package to find the CONFIG class
+#   GENERATOR_PACKAGE = the scala package to find the Generator class in
 #   TB = wrapper over the TestHarness needed to simulate in VCS
```

Comment:
I don't think this is VCS specific, so I would just drop "in VCS" or change it to "to simulate using a verilog simulator"